document/document_en.txt

```diff
@@ -14,50 +14,50 @@ Spin detection:
-	Regular line clears (clearing <=3 lines):
+	Normal line clears (1 to 3 lines):
 		Sends (lines cleared -0.5) attack
 	Special line clears:
 		Spin sends (lines cleared x2) attack,
 		  - B2B sends extra 1/1/2/4/8 (for Spin Single/Double/Triple/Techrash/Techrash+ respectively)
 		  - B2B2B sends (lines cleared x0.5), and +1 extra blocking
-		  - If it's a Mini, x0.25
+		  - Minis reduces the attack to 25% (x0.25 multiplier)
 		Non-Spin Techrash/Techrash+ sends (lines cleared) attack,
-		  - B2B sends extra 1 attack,
-		  - B2B2B sends 50% more attack and +1 extra blocking.
+		  - B2B sends 1 additional line
+		  - B2B2B will have an attack boost of 50% and +1 blocking
 
-	Special line clears will increase B2B gauge, making later special line clears have B2B or B2B2B bonus (see below)
+	Special line clears will the increase B2B gauge, making later special line clears have either a B2B or B2B2B bonus (see below)
 	Half Perfect Clear (a Perfect Clear "with blocks left below". If it's an I clearing 1 line, then the remaining blocks must not be player-placed): Attack +2, Extra Blocking +2
-	Perfect Clear: half all damage above, then +8 to +20 attack (increases within a round by +2 per Perfect Clear) and +2 extra blocking. (note: if lines cleared in this round >4, then B2B gauge will be filled)
+	Perfect Clear (also known as All Clear): half the sum of the damage above, plus additional 8 to 20 attack (increasing by 2 in a single round for each PC) and +2 extra blocking. (note: if lines cleared in this round >4, then B2B gauge will be filled)
 	Combos: All damage above will be given a (combo x25%) bonus, or (combo x15%) if the line clear is a Single, capped at 12 combo. +1 more attack for 3 Combo or more.
 	After calculating all above, the damage value will be rounded down then sent
 
 Score system:
-	The more impressive you play, the more score you get
+	The better you play, the higher the score.
 
 Attack delay:
-	Attack from Doubles/Triples take effect the faster, then Techrash, Spins send rather slow attack, and high combos will send the slowest
-	B2B or B2B2B, while they increase lines sent, they also increase the attack delay. Minis will greatly increase delay.
+	Attacks from Doubles/Triples take effect the faster, followed by Techrash, Spins, which send slow attacks, and high combos are the slowest
+	B2B or B2B2B, while they increase lines sent, they also increase the attack delay. Minis will greatly increase the delay.
 
 Countering:
-	When you send attack, if there is garbage in queue, extra blocking will be used first, then attack, countering the earliest attack at a 1:1 ratio.
-	Unused extra blocking will be discarded, then remaining attack will be sent to your opponent.
+	When you send attacks, if there is garbage in queue, extra blocking will be used first, then attack, countering the earliest attack at a 1:1 ratio.
+	Any extra blocking you didn't use will be discarded, and the last remaining attack power will be sent to your opponent.
```

Comment:
"the last remaining attack power"???
I think you mean something like "**finally**, the remaining attack power will be sent..."